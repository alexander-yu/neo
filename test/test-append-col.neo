void main(){
    create matrix<int> a = [[1,2,5],[3,4,4],[5,6,2]];
    create matrix<int> a1 = [[8], [9], [10]];
    println(iappend_col(a, a1));
    create matrix<float> b = [[1.5,2.1,-0.5],[3.15,4.1,4.1],[5.3,6.0,2.1]];
    create matrix<float> b1 = [[8.5], [9.5], [10.5]];
    println(fappend_col(b, b1));
}