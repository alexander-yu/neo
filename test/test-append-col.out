[[1, 2, 5, 8],
 [3, 4, 4, 9],
 [5, 6, 2, 10]]
[[1.5, 2.1, -0.5, 8.5],
 [3.15, 4.1, 4.1, 9.5],
 [5.3, 6, 2.1, 10.5]]